lib/common.h

```diff
@@ -8,9 +8,10 @@ using std::string;
 using std::vector;
 
 namespace common {
-
-    static const symbol S_SYS = symbol("EOS", 4);
+    static const symbol S_SYS = symbol("SYS", 4);
```

Comment:
Why is this being changed to "SYS"?

Reply:
you are right. It should be chain specific. So for the general contract version, it should still say EOS.